hack/images/dcos/Makefile

```diff
@@ -1,65 +1,153 @@
-mkfile_path     := $(abspath $(lastword $(MAKEFILE_LIST)))
-current_dir     := $(patsubst %/,%,$(dir $(mkfile_path)))
-
-FROM ?= $(current_dir)/_build/k8sm
-TARGET_OBJ = km kubectl
-SOURCE_OBJ = $(TARGET_OBJ:%=$(FROM)/%)
-
+# input variables
+GIT_REF ?= v1.0.2-v0.6.1
+GIT_REPO ?= mesosphere/kubernetes
```

Comment:
We need a way to use an upstream branch. But otherwise this is completely correct. 

We tag on our repo. We will hardly ever use a tag from upstream (only for the first official release of the upstream repo maybe). The usual case will be that we have to pick commits from master (our upstreamed changes) which did make it into the release branch. With the v1.0.2-v0.6.1 package this means, we picked from master:

- the resource accounting 
- the km minion

Both are on master but we missed the release by a few weeks.